### PR TITLE
Update croniter stub to match latest versions

### PR DESCRIPTION
**Pull request by sbv-csis**
In the latest versions of croniter [a new `hash_id`  argument has been added to the `__init__`](https://github.com/kiorky/croniter/blob/master/docs/CHANGES.rst#1012-2021-04-13)

### CODE REVIEW

File: stubs/croniter/METADATA.toml [hunk 1]
```diff
@@ -1,2 +1,2 @@
-version = "0.1"
-python2 = true
+version = "1.0"
+python2 = false
```
Comment (srittau) [inline stubs/croniter/METADATA.toml:2]: According to croniter's `setup.py`, Python 2 is still supported. Any reason to drop it here?

**Review by srittau — COMMENTED** [review]
Looks good to me, but one question below.

**Comment by hauntsaninja** [comment]
(Note that it would be very unexpected if mypy primer did find a difference, because since modular typeshed it does not check changes to third party stubs)

**Comment by JelleZijlstra** [comment]
@sbv-csis are you still interested in moving this forward?

**Comment by Akuli** [comment]
Merged master into this because it conflicted with #5872.

**Comment by srittau** [comment]
I am going to close this for now as there hasn't been any responses to the feedback left. Please reopen it if there is still interest.

**Pull request closed without merging**